src/main/java/net/snowflake/ingest/streaming/SnowflakeStreamingIngestChannel.java

```diff
@@ -82,6 +82,12 @@ public interface SnowflakeStreamingIngestChannel {
   /**
    * Close the channel, this function will make sure all the data in this channel is committed
    *
+   * <p>Note that this call with drop=true will delete <a
+   * href=https://docs.snowflake.com/en/user-guide/data-load-snowpipe-streaming-overview#offset-tokens>Offset
+   * Token</a> and other state from Snowflake servers unless the channel has already been opened in
```

Comment:
> unless the channel has already been opened in another client.

Can you explain this? Even if the channel was open in another client, it would drop it right?
Or was there a check that doesn't allow a non-client owner to drop?